src/main/java/tech/fastj/graphics/display/package-info.java

```diff
@@ -0,0 +1,5 @@
+package tech.fastj.graphics.display;
+
+/**
+ * C
```

Comment:
This seems incomplete -- it should be removed alongside the other empty package infos.